components/thumbnail/metadata/thumbnail.yml

```diff
@@ -3,100 +3,205 @@ status: Contribution
 sections:
   - name: Migration Guide
     description: |
-      ### T-shirt sizing
-      Thumbnail now supports t-shirt sizing with the standard naming convention and requires that you specify the size by adding a `.spectrum-Thumbnail--size*` class.
-
-      Additionally, the available sizes have changed: `XXL` and `XL` have been completely removed, and `XS` and `XXS` have been added.
-
-      | Deprecated Classname | New Classname           |
-      | -------------------- | ----------------------- |
-      | `~.spectrum-Thumbnail--XXL~` | REMOVED |
-      | `~.spectrum-Thumbnail--XL~` | REMOVED |
-      | `~.spectrum-Thumbnail--L~` | `.spectrum-Thumbnail--sizeL` |
-      | `~.spectrum-Thumbnail--M~` | `.spectrum-Thumbnail--sizeM` |
-      | `~.spectrum-Thumbnail--S~` | `.spectrum-Thumbnail--sizeS` |
-      | | `.spectrum-Thumbnail--sizeXS` |
-      | | `.spectrum-Thumbnail--sizeXXS` |
+      ### Thumbnail sizing
+      Thumbnail sizes are specified by appending a number to the size class: `.spectrum-Thumbnail--size*`. Sizes scale expotentially and are based on the Spectrum type scale.  These range from size-50 to size-1000.
 
       ### Image Cover
       Thumbnail now offers a way to control the content inside a child `img` tag by adding a modifier `spectrum-Thumbnail--cover` class in addition to the `spectrum-Thumbnail` class.
 
+      ### Layer
+      Adds variant for when thumbnails are used in layer management (such as the Compact or Detail Layers panels). When used this way the thumbnail is given a thick blue border when selected.
 examples:
   - id: thumbnail-image
-    description: Rectangular images will fill the entire space.
     name: Thumbnail
     markup: |
-      <div class="spectrum-Thumbnail spectrum-Thumbnail--sizeM">
-        <img class="spectrum-Thumbnail-image" src="img/thumbnail.png" alt="Woman crouching">
-      </div>
-  - id: thumbnail-selected
-    name: Thumbnail (selected)
-    markup: |
-      <div class="spectrum-Thumbnail spectrum-Thumbnail--sizeM is-selected">
-        <img class="spectrum-Thumbnail-image" src="img/thumbnail.png" alt="Woman crouching">
+      <div class="spectrum-Thumbnail spectrum-Thumbnail--size500">
+        <div class="spectrum-Thumbnail-image-wrapper">
+          <img class="spectrum-Thumbnail-image"src="img/example-ava.jpg" alt="women resting head in hands">
+        </div>
```

Comment:
@Westbrook  Any concerns about this additional div from a SWC perspective? 

I have added the image-wrapper div to each of the variants to fix an issue with the focus state and the thumbnail image not maintaining border radius. This is an issues currently in place but the new design removed the double border on the focus state making is more visible.

Reply:
Shouldn't be an issue.